src/ReachSets/DiscretePost/DecomposedDiscretePost.jl

```diff
@@ -0,0 +1,143 @@
+export DecomposedDiscretePost
+
+"""
+    DecomposedDiscretePost <: DiscretePost
+
+Textbook implementation of a discrete post operator, but with lazy decomposed intersections.
+
+### Fields
+
+- `options` -- an `Options` structure that holds the algorithm-specific options
+
+### Algorithm
+
+The algorithm is based on [Flowpipe-Guard Intersection for Reachability
+Computations with Support Functions](http://spaceex.imag.fr/sites/default/files/frehser_adhs2012.pdf).
+"""
+struct DecomposedDiscretePost <: DiscretePost
+    options::Options
+
+    function DecomposedDiscretePost(𝑂::Options)
+        𝑂copy = copy(𝑂)
+        # TODO: Check why it takes always default value for convex_hull
+        check_aliases_and_add_default_value!(𝑂.dict, 𝑂copy.dict, [:overapproximation], Hyperrectangle)
```

Comment:
What do you mean here?

Reply:
Somehow user-defined `: overapproximation` doesn't work in clustering function. It always overapproximate `ConvexHull` with `Hyperrectangular`.